Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4891,15 +4968,17 @@ class CommandResults:
     :type mark_as_note: ``bool``
     :param mark_as_note: must be a boolean, default value is False. Used to mark entry as note.
 
+    :type scheduled_command_config: ``Common.ScheduledCommandConfiguration``
+    :param scheduled_command_config: manages the way the command should be polled.
+
     :return: None
     :rtype: ``None``
     """
 
     def __init__(self, outputs_prefix=None, outputs_key_field=None, outputs=None, indicators=None, readable_output=None,
                  raw_response=None, indicators_timeline=None, indicator=None, ignore_auto_extract=False,
-                 mark_as_note=False, polling_command=None, polling_args=None, polling_timeout=None,
-                 polling_next_run=None, relations=None):
-        # type: (str, object, object, list, str, object, IndicatorsTimeline, Common.Indicator, bool, bool,str, dict, str, str, list) -> None # noqa: E501
+                 mark_as_note=False, relations=None, scheduled_command_config=None):
```

Comment:
prefer short names if it still delivers the message.
`scheduled_command_config` -> `scheduled_command`